spec/features/user_deletes_cart_items_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.